libs/wingsdk/src/std/array.ts

```diff
@@ -279,4 +293,17 @@ export class MutArray {
     value;
     throw new Error("Macro");
   }
+
+  /**
+   * Removes first occurence of a given value in an array
+   *
+   * @macro ((obj, args) => { if (obj.indexOf(args[0]) !== -1) { obj.splice(obj.indexOf(args[0]), 1); return true; } return false; })($self$, [$args$])
+   *
+   * @param value the value to remove
+   * @returns true if value was removed
```

Comment:
I wonder if we need to capitalize the two lines above too,
@Chriscbr @MarkMcCulloh @hasanaburayyan  what do you think?

Reply:
I don't have a strong opinion (other than consistency), so right now params/returns comments seem to be lowercase